src/widgets/Header/ui/Icons/Icons.tsx

```diff
@@ -7,11 +7,16 @@ import css from './Icons.module.scss';
 type TIconsProps = object;
 
 const Icons: FC<TIconsProps> = () => {
+  const [cartCount] = useState(1);
+
   return (
     <div className={css.icons}>
       <Icon className={css.icon} to="/login" type="account" />
       <Icon className={css.icon} type="like" />
-      <Icon className={css.icon} type="cart" />
+      <div className={css.iconBtn}>
+        <Icon className={css.icon} type="cart" />
+        <span className={css.cartCount}>{cartCount}</span>
+      </div>
     </div>
```

Comment:
need to think how to make it better with redux store

Reply:
i didnt do header